mistral/pixtral-12b/README.md

```diff
@@ -0,0 +1,81 @@
+# Mixtral 8x7B Instruct Truss
+
+This is a [Truss](https://truss.baseten.co/) for Pixtral 12B. Pixtral 12B parameter language model released by [Mistral AI](https://mistral.ai/) and is a multimodal (text + vision) LLM. This README will walk you through how to deploy this Truss on Baseten to get your own instance of it.
+
+
+## Deployment
+
+First, clone this repository:
+
+```sh
+git clone https://github.com/basetenlabs/truss-examples/
+cd mistral/pixtral-12b
+```
+
+Before deployment:
+
+1. Make sure you have a [Baseten account](https://app.baseten.co/signup) and [API key](https://app.baseten.co/settings/account/api_keys).
+2. Install the latest version of Truss: `pip install --upgrade truss`
+
+With `pixtral-12b` as your working directory, you can deploy the model with:
+
+```sh
+truss push --publish
```

Comment:
since the repo is gated, we should also remind users to accept the ToS on the HF repo, ensure they have an hf token set, and push with --trusted. You can probably copy that part of the README from one of the other examples that require an hf token